.chloggen/support_substring_func.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: pkg/ottl, processor/transform
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add substring factory function, returns a substring from the given start index to the specified length.
```

Comment:
```suggestion
note: Add substring Converter, returns a substring from the given start index to the specified length.
```

Reply:
We are in the process of renaming factory function to converter: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16718